src/main/java/de/minebench/plotgenerator/PlotChunkGenerator.java

```diff
@@ -71,7 +71,7 @@ public ChunkData generateChunkData(World world, Random random, int x, int z, Bio
             startZ = length + startZ;
         }
 
-        int startY = Math.max(world.getMinHeight(), center.getBlockY());
+        int startY = Math.max(-64, center.getBlockY());
```

Comment:
This feels weird. How does that work with worlds that go below -64?

Reply:
Then can just remove the normalization of the data, otherwise it is hard to get the minimum world height.